### PR TITLE
Create UML Sequence Diagram for DeleteMeetingCommand

### PR DESCRIPTION
**Pull request by psh12320**

### CODE REVIEW

File: docs/diagrams/DeleteMeetingSequenceDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+CommandResult --> DeleteMeetingCommand
+deactivate CommandResult
+
+DeleteMeetingCommand --> LogicManager : r
+deactivate DeleteMeetingCommand
+
+[<-- LogicManager
+deactivate LogicManager
+@enduml
+
```
Comment (marraknas) [inline docs/diagrams/DeleteMeetingSequenceDiagram.puml:102]: Works! Nice job!

**Review by marraknas — COMMENTED** [review]
LGTM!

File: docs/diagrams/DeleteMeetingSequenceDiagram.puml (hunk 1 above)
Comment (PeterHW963) [reply docs/diagrams/DeleteMeetingSequenceDiagram.puml:102]: how do you know that it works brother

File: src/main/java/seedu/address/model/AddressBook.java [hunk 2]
```diff
@@ -125,10 +125,22 @@ public void addMeeting(Meeting m) {
         meetings.addMeeting(m);
     }
 
+    public void deleteMeeting(Meeting m) {
```
Comment (PeterHW963) [inline src/main/java/seedu/address/model/AddressBook.java:128]: ah yes, deleting meetings using a 💣

File: src/main/java/seedu/address/model/AddressBook.java [hunk 3]
```diff
@@ -125,10 +125,22 @@ public void addMeeting(Meeting m) {
         meetings.addMeeting(m);
```
Comment (PeterHW963) [inline src/main/java/seedu/address/model/AddressBook.java:125]: If i join isys can i also be added to the meetings 🥺

**Review by PeterHW963 — COMMENTED** [review]
LGTM! Don't mind my comments

File: docs/diagrams/DeleteMeetingSequenceDiagram.puml (hunk 1 above)
Comment (ulricolo7) [reply docs/diagrams/DeleteMeetingSequenceDiagram.puml:102]: it earns money for the family and always busy from 9-5

**Review by ulricolo7 — COMMENTED** [review]
LGTM! Is Peter drunk?

**Pull request merged**